src/main/java/WordFinder.java

```diff
@@ -1,30 +1,37 @@
 import java.io.IOException;
+import java.net.URISyntaxException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.*;
 import java.util.stream.Collectors;
 
 class WordFinder {
 
-    private String path = "/Users/jason.motee/Exercises/coundown-helper-app/src/main/resources/words_alpha.txt";
+    private String path;
 
-    void setPath(String path) {
+    WordFinder() {
+        this("words_alpha.txt");
+    }
+
+    public WordFinder(String path) {
         this.path = path;
     }
 
-    List<String> search(String target) throws IOException {
-        List<String> foundWords = Files.lines(Paths.get(path))
+    List<String> search(String target) throws IOException, URISyntaxException {
+        List<String> foundWords = Files.lines(Paths.get(ClassLoader.getSystemResource(path).toURI()))
                 .map(dictionaryWord -> findWord(dictionaryWord, target))
                 .filter(Objects::nonNull)
-                .sorted(new StringLengthComparator())
+                .sorted(Comparator.comparing(String::length).reversed())
                 .collect(Collectors.toList());
 
+        //do we really need to process the stream twice?
         return foundWords.stream()
                 .filter(words -> words.length() == foundWords.get(0).length())
                 .collect(Collectors.toList());
     }
 
     private String findWord(String dictionaryWord, String wordJumble) {
+        //gratuitous streamage?
         List<String> splitDictionaryWord = Arrays.stream(dictionaryWord.split("")).collect(Collectors.toList());
```

Comment:
Initially I thought these seems a little over-the-top but having thought about it, maybe there isn't a nice way to do the conversion.  Could we use a character type rather than Strings though?